provider/ec2/environ.go

```diff
@@ -1909,6 +1909,8 @@ func (e *environ) allModelVolumes(ctx context.ProviderCallContext, includeRootDi
 	return listVolumes(e.ec2Client, ctx, includeRootDisks, makeModelFilter(e.uuid()))
 }
 
+var permissionDescription = aws.String("juju group ingress rule")
```

Comment:
Not sure I agree with this wording for the following reasons.

We have no concept of group in Juju this is an AWS term and it's parent is a group so that is inferred.

The description is also not pertaining to a rule but more to source ip addresses.

Would a better string be `"juju defined source ip range %s", sourceIp's`

Reply:
This is probably an improvement. 

Not convinced reference to group is a problem, because the description of sec groups created by Juju is `juju group`

Also, some rules are open to sources ips, some to sources which own the sec groups

Reply:
Also, the description is pertaining more to the rule than the ip source range, it just looks like it isn't because each rule only has one 1 CIDR, but the data structure here does. As a result, the description lives in the ip ranges, when really it applies to the whole run

Reply:
I was also considering adding `juju defined` as you suggest, but we have very few characters before aws tends to truncate the description, so I think it should be left out